.cbmc-batch/jobs/aws_cryptosdk_decrypt_body/aws_cryptosdk_decrypt_body_harness.c

```diff
@@ -41,7 +41,8 @@ void aws_cryptosdk_decrypt_body_harness() {
 
     struct content_key *content_key;
 
-    uint8_t *tag = can_fail_malloc(props->tag_len);
+    uint8_t *tag = malloc(props->tag_len);
+    // possible that more memory could be allocated, but need to be able to write tag_len bytes to buffer tag.
```

Comment:
Let's avoid C++ comments. Repeat this pattern in the other files as well.
```c
/*
 * Possible that more memory could be allocated, but need
 * to be able to write tag_len bytes to buffer tag.
 */
uint8_t *tag = malloc(props->tag_len);
```
Also, what do you mean by "but need to be able to write tag_len bytes to buffer tag"? Could you clarify?

Reply:
Yes, I've made the comment clearer and changed the style as suggested.